app/services/application_service.rb

```diff
@@ -0,0 +1,5 @@
+class ApplicationService
+  def self.perform(*args, &block)
+    new(*args, &block).perform
+  end
+end
```

Comment:
this pattern is also present in the OKC blotter code but since that's not merged in I'm adding it here too